mifospay/src/main/res/layout/activity_login.xml

```diff
@@ -12,31 +12,17 @@
 
     <ImageView
         android:id="@+id/image_logo"
-        android:layout_width="300dp"
-        android:layout_height="160dp"
-        android:src="@drawable/mifos_icon"
```

Comment:
@Bhavnaharitsa we are not using `mifos_icon` anywhere else, so can delete this drawable also

Reply:
done